### PR TITLE
Move document from g/documentation

### PR DESCRIPTION
**Pull request by Kristian-ZH**
**How to categorize this PR?**

/area documentation
/kind enhancement
/platform gcp

**What this PR does / why we need it**:
Move document from https://github.com/gardener/documentation/tree/master/website/documentation/guides/administer_shoots/gardener_gcp because the documents there are not maintained.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
None
```

### CODE REVIEW

File: docs/tutorials/kubernetes-cluster-in-gpc-with-gardener/_index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
```
Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-in-gpc-with-gardener/_index.md:1]: Type in the dir name - `kubernetes-cluster-in-gpc-with-gardener` should be `kubernetes-cluster-in-gcp-with-gardener/_index.md`.

```
mv ./docs/tutorials/kubernetes-cluster-in-gpc-with-gardener ./docs/tutorials/kubernetes-cluster-in-gcp-with-gardener/
```

File: docs/tutorials/kubernetes-cluster-in-gpc-with-gardener/_index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+10. On tab *INFRASTRUCTURE*, choose the secret you created before. 
+
+
+    <img src="images/gcpcreatecluster1.jpg">
+
+    <img src="images/gcpcreatecluster2.jpg">
+
+    <img src="images/create_cluster4.jpg">
+
+    <img src="images/gcpcreatecluster2.jpg">
```
Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-in-gpc-with-gardener/_index.md:66]: I think it should be

```suggestion
    <img src="images/gcpcreatecluster3.jpg">
```

Currently gcpcreatecluster2.jpg is present 2 times and gcpcreatecluster3.jpg is not present in the guide at all.

File: docs/tutorials/kubernetes-cluster-in-gpc-with-gardener/_index.md (hunk 1 above)
Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-in-gpc-with-gardener/_index.md:1]: I don't see `images/copy_icon.png` being linked/used by this tutorial. Can we remove it or make the tutorial use it?

**Review by ialidzhikov — APPROVED** [review]
/lgtm

File: docs/tutorials/kubernetes-cluster-in-gcp-with-gardener/_index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+3. [Create a new service account in GCP](https://console.cloud.google.com/iam-admin/serviceaccounts) and assign the roles required by Gardener.
+
+    <img src="images/gcpcreateserviceaccount0.jpg">
+
+    <img src="images/gcpcreateserviceaccount1.jpg">
+
+4. To create a key for the service account, choose *Actions* and then *Create key*.
+
+    <img src="images/gcpcreatekey.jpg">
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-gcp-with-gardener/_index.md:31]: Same here. Refer the gcp docs. https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys

File: docs/tutorials/kubernetes-cluster-in-gcp-with-gardener/_index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+5. Save the private key of the service account in JSON format.
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-gcp-with-gardener/_index.md:33]: This section can be combined with `4.` as the step is described in https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys

File: docs/tutorials/kubernetes-cluster-in-gcp-with-gardener/_index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    > Note: Save the key of the user, it’s used later to create secrets for Gardener.
+
+6. [Enable the Google compute API](https://console.developers.google.com/apis/library/compute.googleapis.com).
+
+    <img src="images/gcpcomputeengineapi.jpg">
+
+
+7. [Enable the Google IAM API](https://console.developers.google.com/apis/api/iam.googleapis.com/overview).
+    <img src="images/gcpiamapi.jpg">
+
```
Comment (dkistner) [inline docs/tutorials/kubernetes-cluster-in-gcp-with-gardener/_index.md:45]:
```suggestion
6. Enable APIs
  - [Compute API](https://console.developers.google.com/apis/library/compute.googleapis.com)
  - [IAM API](https://console.developers.google.com/apis/api/iam.googleapis.com/overview)
```

The respective figures can be dropped.

**Review by dkistner — COMMENTED** [review]
Also like mentioned in https://github.com/gardener/gardener-extension-provider-azure/pull/379 many of the figures are outdated. We might want to update them?

I would also motivate to use where ever possible the gcp docs.
Otherwise this guide will probably be soon outdated again.

**Comment by ialidzhikov** [comment]
> I will try to update the structure & content a bit afterwards.

Thank you. I created an umbrella issue on g/documentation side - https://github.com/gardener/documentation/issues/287. If you have more suggestions what to improve, let's add them to the issue description.

**Pull request merged**